### PR TITLE
Bump the official supported Unity version to 18.4

### PR DESCRIPTION
**Pull request by surfnerd**

### CODE REVIEW

File: UnitySDK/Packages/manifest.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "com.unity.modules.unityanalytics": "1.0.0",
+    "com.unity.modules.unitywebrequest": "1.0.0",
+    "com.unity.modules.unitywebrequestassetbundle": "1.0.0",
+    "com.unity.modules.unitywebrequestaudio": "1.0.0",
+    "com.unity.modules.unitywebrequesttexture": "1.0.0",
+    "com.unity.modules.unitywebrequestwww": "1.0.0",
+    "com.unity.modules.vehicles": "1.0.0",
+    "com.unity.modules.video": "1.0.0",
+    "com.unity.modules.vr": "1.0.0",
+    "com.unity.modules.wind": "1.0.0",
```
Comment (chriselion) [inline UnitySDK/Packages/manifest.json]: Do we really need all these?

Comment (surfnerd) [reply UnitySDK/Packages/manifest.json]: It failed without them

Comment (surfnerd) [reply UnitySDK/Packages/manifest.json]: I can try to trim it down manually if you want me to.  This is what the editor put in by default.  Aren't you on PTO? :)

**Comment by surfnerd** [comment]
> Guess we can get rid of the 2017.4 yamato tests now :)

Doing that now :)

File: UnitySDK/Packages/manifest.json (hunk 1 above)
Comment (chriselion) [reply UnitySDK/Packages/manifest.json]: Just seems weird that we need wind and particles. But maybe they're transitive dependencies from other packages that we do need.

**Pull request merged**